engineering/workflow/default-workflow.md

```diff
@@ -60,29 +60,148 @@ Last day of Iteration
 :::
 
 (meetings:backlog-refinement)=
+
 ### 1. Pre-planning and refinement
 
-The Refinement meetings prepares the Iteration board for the Iteration planning
-meetings, which relies on having sequenced and prioritized tasks to be available in the board's Refined column.
+Tasks in the backlog are continually added and refined asynchronously by the whole team, ready for Iteration planning meetings, which rely on having sequenced and prioritized tasks to be available in the board's Refined column.
```

Comment:
Is anybody allowed to define prioritization? AKA is anybody encouraged to move any card to any ordering in the board? Are there situations where this needs to be done by a specific person? (e.g. the product manager has the final say if there are disagreements on priority?)